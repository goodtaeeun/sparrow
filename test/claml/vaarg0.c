__builtin_va_list a;
b() { *__builtin_va_arg(a, int *); }
