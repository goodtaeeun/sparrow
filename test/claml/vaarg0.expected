/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "vaarg0.c"
__builtin_va_list a;
#line 2 "vaarg0.c"
int b(void) {
  int *__cil_tmp1;

  { return (0); }
}
